microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// New creates a configured storage ready to be used in tests.
+func New() microstorage.Storage {
```

Comment:
No. It has no input, and it's tested that it doesn't.

`Must` is used when function has input. See regexp or templates.